### ISSUE TITLE
What is the reason for the difference in distribution between the validation set and the training set?

### ISSUE THREAD

**Issue by Canbe0220** [thxiwilldoit/DOAGNN#2]
The process time ranges from [1,100] for the validation set, while it ranges from [1,20]  for the training set. Why has it been set up this way?

**Comment by thxiwilldoit** [comment]
Similar to SD2 DATASET in DANIEL, attempt to improve its generalization ability.

**Comment by thxiwilldoit** [comment]
Sorry for only seeing this message now :(
